BE/src/main/java/kr/codesquad/sidedish/service/OrderService.java

```diff
@@ -0,0 +1,36 @@
+package kr.codesquad.sidedish.service;
+
+import kr.codesquad.sidedish.domain.Item;
+import kr.codesquad.sidedish.domain.Order;
+import kr.codesquad.sidedish.repository.ItemRepository;
+import kr.codesquad.sidedish.repository.OrderRepository;
+import kr.codesquad.sidedish.web.dto.order.OrderRequest;
+import kr.codesquad.sidedish.web.dto.order.OrderResponse;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class OrderService {
+
+    private final OrderRepository orderRepository;
+    private final ItemRepository itemRepository;
+
+    public OrderService(OrderRepository orderRepository, ItemRepository itemRepository) {
+        this.orderRepository = orderRepository;
+        this.itemRepository = itemRepository;
+    }
+
+    public OrderResponse create(OrderRequest orderRequest) {
+        Order saved = orderRepository.save(orderRequest.toEntity());
+        Integer quantity = orderRequest.getQuantity();
+        Long itemId = orderRequest.getItemId();
+        Optional<Item> orderItem = itemRepository.findById(itemId);
+        Item item = orderItem.orElseThrow();
```

Comment:
굳이 분리해서 사용 할 필요가 없어보여요.

```suggestion
        Item item = itemRepository.findById(itemId).orElseThrow(RuntimeException::new);
```

Reply:
네 코멘트 주신대로 코드수정 했습니다 :)